RELEASE_NOTES.md

```diff
@@ -1,23 +1,22 @@
 # ChefDK 1.2 Release Notes
 
+## delivery CLI
 - Updated the project.toml file generator with:
   - A `functional` phase that will be optional for implementation.
   - New `remote_file` option to specify a remote project.toml.
-- Bundle the Stunnel tool and FIPS OpenSSL module for users who need to enforce FIPS compliance.
-
-- Add Chef Server policyfile resolution.
-  - Adds a `chef_server` default source option to Policyfiles
-
-### delivery-cli Changes
-
-- New `remote_file` option to load `project.toml` from URL.
```

Comment:
I wanted to mention it once, and even thoughts is part of the CLI, the CLI is part of the DK so I'm okay with it.